### PR TITLE
Hacktober 2020

### PR DESCRIPTION
**Pull request by dahiya-code**
I have made changes which I felt are enough. I would be happy to hear your suggestions :)

### CODE REVIEW

**Comment by annu12340** [comment]
Pls resolve the conflicts

File: README.md [hunk 1]
```diff
@@ -1,70 +1,132 @@
...
+* Run :
+```
+python3 main.py
+```
+ 
+* Copy the localhost url (usually localhost:5000/) and paste in browser
 
-<hr>  
+<!-- USAGE EXAMPLES -->
+## Usage
```
Comment (annu12340) [inline README.md]: Change **usage** to **Demo** (or **screenshots**)

File: README.md [hunk 2]
```diff
@@ -1,70 +1,132 @@
-# Choose-Smart
-As consumers, we face a great dilemma in choosing the right product from a wide variety of goods. This application helps the user to analyse the product and arrive at a smarter choice .
+<br />
+  <h1 align="center">Choose-Smart🖥️</h3>
+
+  <p align="center">
+    Always confused about where to get the best deals on your favourite things? Don't worry Choose-Smart has come to your rescue
+    <br />
+    <a href="https://github.com/annu12340/Choose-Smart"><strong>Explore the docs »</strong></a>
```
Comment (annu12340) [inline README.md]: We don't require this line (This link is just redundant)

File: README.md [hunk 3]
```diff
@@ -1,70 +1,132 @@
...
 
-- Provides insight to the consumer about the hazardous effects of the ingredients
+* Provides insight to the consumer about the hazardous effects of the ingredients
+
+* Suggest the best deal avaiable on popular online shopping platform
+
+* Sentiment analysis of reviews of a product using Twitter API
+
+* Displays all the important laws that a consumer must be aware of
+
```
Comment (annu12340) [inline README.md:43]: Pls add the screenshots here (instead of keeping it at the bottom)

File: README.md [hunk 4]
```diff
@@ -1,70 +1,132 @@
...
 ![9](https://user-images.githubusercontent.com/43414928/80920734-6aced100-8d8f-11ea-9f9a-357adff47b8f.png)
 
 
-Different consumer laws
+#### Different consumer laws 📜
 ![10](https://user-images.githubusercontent.com/43414928/80920742-70c4b200-8d8f-11ea-9c63-091a19f84d62.png)
 
 
+<!-- ROADMAP -->
+## Roadmap
```
Comment (annu12340) [inline README.md]: I think the roadmaps is not required

**Review by annu12340 — CHANGES_REQUESTED** [review]

**Comment by dahiya-code** [comment]
Sir please help me in HACKTOBER FEST 2020
We will honor all valid pull requests prior to this change, and as of October 3, 2020 at 12:00:00 UTC – and October 3 in all time zones – pull requests will only count toward earning a T-shirt or planting a tree if they are submitted in a repository classified with the ‘hacktoberfest’ topic. The pull requests will also need to be merged, approved by a maintainer, or labeled as ‘hacktoberfest-accepted’ in order to qualify. The deadline for completions, merging, labeling, and approving is November 1.
Latest guidelines issued by the organizers
I sincerely hope you will help me
I have already submitted a pull request
https://hacktoberfest.digitalocean.com/hacktoberfest-update
I am attaching the link too for your convenience

File: README.md [hunk 5]
```diff
@@ -55,6 +74,7 @@ This section should list any major frameworks that you built your project using.
 
 ### Prerequisites
```
Comment (annu12340) [inline README.md]: Change Prerequisites to installation
and remove the title ** installation** from line 73

**Review by annu12340 — CHANGES_REQUESTED** [review]

**Comment by annu12340** [comment]
Pls resolve the conflicts and then I would merge the pr :)

**Comment by dahiya-code** [comment]
Done

**Comment by annu12340** [comment]
> Done

Still, the files are showing conflicts
Kindly resolve them

**Comment by annu12340** [comment]
Thank you @navya252001 for the pr 😄

**Comment by dahiya-code** [comment]
thankyou for the opportunity

**Comment by dahiya-code** [comment]
Ma'am please make the changes according to these guidelines for making my PR count
We will honor all valid pull requests prior to this change, and as of October 3, 2020 at 12:00:00 UTC – and October 3 in all time zones – pull requests will only count toward earning a T-shirt or planting a tree if they are submitted in a repository classified with the ‘hacktoberfest’ topic. The pull requests will also need to be merged, approved by a maintainer, or labeled as ‘hacktoberfest-accepted’ in order to qualify. The deadline for completions, merging, labeling, and approving is November 1.

**Comment by dahiya-code** [comment]
Label your repository with hacktoberfest
My PR as hacktoberfest-accepted
PLEASE DO IT ASAP

**Comment by dahiya-code** [comment]
Ok its cool I just saw your issue is hacktober marked

**Comment by annu12340** [comment]
> Ok its cool I just saw your issue is hacktober marked

Everything is sorted now, right?
Any changes required from my side?

**Comment by dahiya-code** [comment]
Yes thanku

**Pull request merged**